dashboard/apps/prototype_admin/static/prototype_admin/js/dropdown_filter.js

```diff
@@ -0,0 +1,37 @@
+let listVisible = false;
```

Comment:
this might need to be `var` because there is no transpiler to convert it to es5.